templates/index.md.tmpl

```diff
@@ -81,4 +81,32 @@ Using this flow is less secure since the token is long-lived. You can provide th
 2. Setting the environment variable `STACKIT_SERVICE_ACCOUNT_TOKEN`
 3. Setting it in the credentials file (see above)
 
+# Backend configuration
+
+To keep track of your terraform state, you can configure an [S3 backend](https://developer.hashicorp.com/terraform/language/settings/backends/s3) using [STACKIT Object Storage](https://docs.stackit.cloud/stackit/en/object-storage-s3-compatible-71009778.html).
+
+To do so, you need an Object Storage [S3 bucket](https://docs.stackit.cloud/stackit/en/basic-concept-objectstorage-71009785.html#BasicConceptObjectStorage-Buckets) and [credentials](https://docs.stackit.cloud/stackit/en/basic-concept-objectstorage-71009785.html#BasicConceptObjectStorage-Credentials) to access it. If you need to create them, check [Getting Started - Object Storage](https://docs.stackit.cloud/stackit/en/getting-started-objectstorage-71009792.html).
+
+Once you have everything setup, you can configure the backend by adding the following block to your terraform configuration:
+
+```
+terraform {
+  backend "s3" {
+    bucket = "BUCKET_NAME"
+    key    = "path/to/key"
```

Comment:
```suggestion
    key    = "PATH/TO/KEY"
```

Reply:
we have other paths above that are also user-input and have lower-case. I think in this case it looks a bit strange to have them as upper case because they are a path. And from the value it looks clear that it should be replaced with the actual path to the key

For consistency in EVERY user-input then we would have to review all of the documentation, because I think there are some differences throughout the README